Makefile

```
@@ -6,16 +6,35 @@ DEPLOY_MODE ?= helm
+	helm dependency build charts/osac/
+
+.PHONY: dev-deps
+dev-deps: ## Build chart deps from local submodules (for development)
+	git submodule update --init --recursive
+	cp charts/osac/Chart.yaml charts/osac/Chart.yaml.bak
+	./scripts/rewrite-chart-deps-local.sh && \
+		helm dependency build charts/osac/; \
+		rc=$$?; mv charts/osac/Chart.yaml.bak charts/osac/Chart.yaml; exit $$rc
+
+.PHONY: bump-chart
+bump-chart: ## Bump a subchart version: make bump-chart BUMP_CHART=fulfillment-service BUMP_VERSION=0.0.68
```

checkmake (0.3.2)

[warning] Target body for "bump-chart" exceeds allowed length of 5 lines (6).

(maxbodylength)